modules/nbde-openshift-installation-with-nbde.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// security/nbde-implementation-guide.adoc
+
+[id="nbde-openshift-installation-with-nbde_{context}"]
+= OpenShift installation with NBDE
+
+Network-Bound Disk Encryption (NBDE) must be enabled when an OpenShift node is installed. However, you can change the disk encryption policy at any time after it was initialized at installation.
+
+For more information on enabling Tang mode disk encryption during an {product-title} (OCP) installation see: https://docs.openshift.com/container-platform/4.7/installing/install_config/installing-customizing.html#installation-special-config-encrypt-disk-tang_installing-customizing[Enabling Tang disk encryption].
```

Comment:
@StephenJamesSmith just noticed that this is a reference to the 4.7 document! I think this needs to be changed to 4.8.

Is there a better way for it to dynamically link to the version to which this document corresponds?

Reply:
Updated the link to 4.8. Jim- We'll discuss this.